.github/workflows/pull_requests.yaml

```diff
@@ -30,4 +30,6 @@ jobs:
             ${{ runner.os }}-maven-
 
       - name: Build native executable
-        run: mvn package -Pnative
+        run: |
+          mvn package -Pnative -DexcludedGroups=integration-native
+          mvn test -Dquarkus.test.profile=integration-native -Dgroups=integration-native
```

Comment:
Do we need to define both groups and quarkus test profile? What plugin does the groups property map to?

Reply:
The `groups` is JUnit stuff. The issue was that I only wanted to run the native tests on the second pass, but we would need to annotate the non-native tests. At least I didn't find anything ready for quarkus, to just ignore by the profile or something :disappointed:.

Another approach is to use the `ExecutionCondition` as we do in db-syncer.